src/lib.rs

```diff
@@ -116,6 +116,11 @@ pub enum Workload {
         /// Starting number of connections
         nconnections: u32,
 
+        // How many connections to make to the same server address and port with
+        // different client ports
```

Comment:
[nit] I think the naming needs adjustment a little since the current name is a bit indirect. Perhaps `conns_per_addr` or `conns_per_ip`? We don't strictly care that the client port changes in order to achieve this end, but we do care that there will be some number of connections for each IP address the client is using.

Reply:
I have changed `ports_per_addr` to `conns_per_addr`